test/helpers/cilium.go

```diff
@@ -333,6 +349,96 @@ func (c *Cilium) ReportFailed(commands ...string) {
 		fmt.Fprint(wr, res.Output())
 	}
 	fmt.Fprint(wr, "StackTrace Ends\n")
+	c.ReportFailedFull()
+}
+
+//ReportFailedFull runs a variety of commands and write the results to
+//testResultsPath
+func (c *Cilium) ReportFailedFull() {
+	reportCommands := map[string]string{
+		"sudo cilium endpoint list -o json": "endpoint_list_txt",
+		"sudo cilium service list -o json":  "service_list.txt",
+		"sudo cilium config":                "config.txt",
+		"sudo cilium bpf lb list":           "bpf_lb_list.txt",
+		"sudo cilium bpf ct list global":    "bpf_ct_list.txt",
+		"sudo cilium bpf tunnel list":       "bpf_tunnel_list.txt",
+		"sudo cilium policy get":            "policy_get.txt",
+		"sudo cilium status":                "status.txt",
+	}
+
+	reportEndpointsCommands := map[string]string{
+		"cilium endpoint get %s":    "endpoint_%s.txt",
+		"cilium bpf policy list %s": "bpf_policy_list_%s.txt",
+	}
+
+	testPath, err := ReportDirectory()
+	if err != nil {
+		c.logCxt.Errorf("Can't create Test results path '%s' err '%s'", testPath, err)
+		return
+	}
+	reportMap(testPath, reportCommands, c.Node)
+
+	//Endpoint specific information:
+	eps, err := c.GetEndpointsIds()
+	if err != nil {
+		c.logCxt.Errorf("Can not get endpoint ids")
+	}
+	for _, ep := range eps {
+		for cmd, logfile := range reportEndpointsCommands {
+			command := fmt.Sprintf(cmd, ep)
+			res := c.Node.Exec(fmt.Sprintf("sudo %s", cmd))
```

Comment:
Why are these commands executed with sudo? The vagrant user which runs these commands should be able to run Cilium CLI commands without the need to prepend them with sudo. The exception here is any command with bpf in it, which does require root privileges.